src/SourceBuild/content/Directory.Build.props

```diff
@@ -6,6 +6,18 @@
     <DotNetBuildOrchestrator>true</DotNetBuildOrchestrator>
   </PropertyGroup>
 
+  <PropertyGroup Label="ShortStacks">
+    <_OverrideTargetOS>$(OverrideTargetRid.Substring(0, $(OverrideTargetRid.LastIndexOf('-'))))</_OverrideTargetOS>
+    <ShortStack Condition="'$(_OverrideTargetOS)' == 'wasi'">true</ShortStack>
```

Comment:
I wonder if the property name should indicate that this is a "runtime short stack" build.